src/components/tables/RowActions/Delete/Confirmation.tsx

```diff
@@ -0,0 +1,26 @@
+import { Alert, AlertTitle, Box, Stack } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+
+interface DeleteConfirmationprops {
+    deleting: any; //SelectableTableStore['selected'];
```

Comment:
My review began from the bottom up. Similar to the comment left in [src/components/tables/RowActions/Delete/ProgressDialog.tsx](https://github.com/estuary/ui/pull/148/files#diff-658bd9e0df8535c9d44521fc68326dc8659db3e61edc08069e0a396da7bdd31b), curious if `deleteQueue` is a reasonable property name alternative.

Reply:
That's interesting... as even with interfaces and types I naturally just look at the component to see how things are used.